consensus/leader_election/module.go

```diff
@@ -16,13 +16,18 @@ type LeaderElectionModule interface {
 	ElectNextLeader(*typesCons.HotstuffMessage) (typesCons.NodeId, error)
 }
 
-var _ leaderElectionModule = leaderElectionModule{}
+var _ LeaderElectionModule = &leaderElectionModule{}
 
 type leaderElectionModule struct {
 	bus modules.Bus
 }
 
-func Create(_ *typesCons.ConsensusConfig, _ *typesCons.ConsensusGenesisState) (LeaderElectionModule, error) {
+func Create(runtime modules.Runtime) (modules.Module, error) {
```

Comment:
This create wrapper seems like a clunkier design, considering the function signatures are the same.

Have we considered `new(consensusModule).Create(runtime)` instead of using a wrapper for each?

Reply:
🤦‍♂️ you are totally right

https://github.com/pokt-network/pocket/pull/235/files#diff-eba95a8b904f8186683d2f9c106a3dc021f014918420e6a074d0a2b4e59c261dR66-R68

Reply:
Without the wrapper we would have to expose all the structs and actually in this round of changes, I made all modules structs unexported